dts/arm/st/l1/stm32l1.dtsi

```diff
@@ -17,6 +17,10 @@
 #include <freq.h>
 
 / {
+	aliases {
+		die-temp0 = &die_temp;
+	};
```

Comment:
According to `samples/sensor/die_temp_polling/sample.yaml`, sample will be built/tested on all boards that have `die-temp0` alias. With this change, this will be all L1 based boards.
Problem is that not all L1 based boards are configured to be able to run this test, and will likely fail or break.

This addition should be done per board, not at soc level.

Reply:
I got it. I move all changes into boards/arm/nucleo_l152re/nucleo_l152re.dts.